src/categorical_algebra/CSets.jl

```diff
@@ -2,11 +2,12 @@
 """
 module CSets
 export ACSetTransformation, CSetTransformation, components, force, is_natural,
-  migrate!
+  migrate!, serialize, deserialize
```

Comment:
The names `serialize` and `deserialize` are very generic. I suggest that we following the naming convention established for DWD serialization, e.g. for [JSON](https://github.com/AlgebraicJulia/Catlab.jl/blob/46602c6b64367993446212197a850656a7ce8c56/src/wiring_diagrams/JSON.jl#L18). So we would have `parse_json_acset` and `generate_json_acset` and optionally also `read_json_acset` and `write_json_acset` as convenience functions for file-based IO.

Reply:
renamed in latest commit